### PR TITLE
feat: add TechLekh website

### PR DESCRIPTION
**Pull request by drishyakarki**
## What kind of change does this PR introduce? 

- [ ] Bug Fix
- [ ] Documentation change
- [x] Added new company
- [ ] Other(s)

### CODE REVIEW

File: data/companies.json [hunk 1]
```diff
@@ -496,6 +496,22 @@
         "youtube": "https://www.youtube.com/channel/UCNG0LqHlyecBNCcIAnAMtXw"
       }
     },
+    {
+      "name": "TechLekh Services Pvt. Ltd.",
+      "establishedAt": 1420099200,
+      "location": "Sukedhara, Kathmandu",
+      "website": "https://techlekh.com/",
+      "email": "contact@techlekh.com",
+      "phone": "+977-9813953967",
```
Comment (thenishantsapkota) [inline data/companies.json:505]: Does this place have a landline number?
If yes, please provide the landline number.

File: data/companies.json [hunk 2]
```diff
@@ -496,6 +496,22 @@
         "youtube": "https://www.youtube.com/channel/UCNG0LqHlyecBNCcIAnAMtXw"
       }
     },
+    {
+      "name": "TechLekh Services Pvt. Ltd.",
+      "establishedAt": 1420099200,
```
Comment (thenishantsapkota) [inline data/companies.json:501]: Have you checked the authenticity of this?

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: data/companies.json (hunk 1 above)
Comment (drishyakarki) [reply data/companies.json:505]: No, I didn't find one!

File: data/companies.json (hunk 2 above)
Comment (drishyakarki) [reply data/companies.json:501]: yes

**Review by thenishantsapkota — APPROVED** [review]
LGTM!

**Comment by thenishantsapkota** [comment]
Thank you for your contribution @drishyakarki!

**Pull request merged**